pkg/operators/v1/olmconfig_types.go

```diff
@@ -0,0 +1,71 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	DisableCopiedCSVsConditionType = "DisableCopiedCSVs"
```

Comment:
Nit: Should this be past tense so that it reads a little more like some state has (or hasn't) been achieved. Not sure if this was captured in the EP. If so, disregard. E.g.:
```yaml
conditions:
- type: DisabledCopiedCSVs
  status: True
```

Or:

```yaml
conditions:
- type: DisabledCopiedCSVs
  status: False
  reason: DeleteCSVFailure
  message: 'delete csv "foo" in namespace "bar": not permitted'
```